meetings/2025-05-15.md

```diff
@@ -0,0 +1,84 @@
+
+### 4.1.1 File Systems & FormData
+
+It isn’t yet clear what the best approach will be to accept FormData. Andreu has added [a TODO to the Minimum Common API specification](https://min-common-api.proposal.wintertc.org/#issue-92f53c35) and [an issue in the MCA proposal](https://github.com/WinterTC55/proposal-minimum-common-api/issues/63) to discuss more broadly. The catch is that WP tests typically require creating an HTMLFormElement in order to test the FormData, which server-side runtimes would not be expected to support. Some recent changes to WPT do appear to provide, if not a 1:1 test, at least code that can reliably be reused by MCA testing.
+
+Constructor tests are challenging because existing runtimes already have their own behaviour depending on whether zero, one, or two arguments (un)defined.
+
+### 4.1.2 Event loops
+
+Due to a number of conflicting assumptions made about any given runtime’s event loops, not every server runtime is consistent in event management. **Andreu Botella** will go through the WPT and Fetch spec in finer detail between meetings and bring them on a future agenda.
+
+### 4.1.3 JS Tests
+
+There are a number of tests against the ECMAScript language itself, not just the Web Platform; **Andreu Botella** will research whether those were meant to be implemented in Test262.
+
+## 4.2 Additions to Minimum Common API spec
+
+### 4.2.1 Observable API
+
+Chrome is now shipping Observable, so TC55 has run out of time to start making decisions about its place in the Minimum Common API. It’s a significant API to implement, and relies once again on a predictable event loop.
+
+### 4.3 Sanitizer API
+
+On the browser side, the Sanitizer API’s intended purpose is to prevent cross-site scripting vulnerabilities—making it clear to developers identifying “safe” and “unsafe” markup. It will work with ECMAScript TrustedTypes. It *may* be meaningful to developers working on server runtimes, but would require implementing DOM APIs. **Andreu Botella** & **Owen Buckley** will try to bring **J.S. Choi (invited expert)** in to participate in a future meeting to discuss.
+
+## 4.3 Continuing on review of Fetch API
+
+The committee continued through the Fetch API, with **Andreu Botella** updating [the tracking issue](https://github.com/WinterTC55/fetch-workstream/issues/1#issuecomment-2884343435). The review of HTTP-redirect was completed, and review of HTTP-network-or-cache was started, to be finished at a future meeting.
+
+## 5 Any other business
+
+The TC55 organization name has been updated to [WinterTC55](https://github.com/WinterTC55/), and the website theme is being updated to align better with Ecma orange #fc7c00.
+
+## 6 Next meetings
+
+The next meetings will take place as Google Meet video calls fortnightly. The next scheduled is on 29 May, 2025 at 16:30 CEST.
```

Comment:
```suggestion
The next meetings will take place as Google Meet video calls fortnightly. The next scheduled is on 29 May, 2025 at 16:30 CEST. This meeting overlaps with the TC39 plenary, however, so it might need to be cancelled depending on the TC39 scheduling.
```